### DISCUSSION TITLE
FIREFLY: SOMETHING WHEN WRONG! / NETWORK DISCONNECT

### DISCUSSION THREAD

**Discussion by b05-olaniyi** [iotaledger/firefly#2417] [Help & Support]
Hi, Please what can I do now. I later discover that I formatted my PC, so in the process of Setting up back my Firefly That's where I got stocked. Then later someone makes me know that as I format my Pc the IMEI will change.

What is the way out

**Comment by mav2022** [comment]
the same problem accourt with me. No staking in reason that the node was down !

**Reply by MiKeF05** [reply]
The staking is further active and the node is not down. It seems that you can´t connect to the node. Can you share the error log of Firefly? Do you use a VPN or special firewall?

**Reply by b05-olaniyi** [reply]
I'm just tired of everything like this. Just because I formated my PC 🥺. There is nothing like VPN or Firewall on my laptop

**Comment by mav2022** [comment]
Hope you´ll find a solution, because this Problem apeers oftener…. So far, merci
Martin

Gesendet von Mail für Windows

Von: MiKeF05
Gesendet: Mittwoch, 23. Februar 2022 19:20
An: iotaledger/firefly
Cc: mav2022; Comment
Betreff: Re: [iotaledger/firefly] FIREFLY: SOMETHING WHEN WRONG! / NETWORKDISCONNECT (Discussion #2417)

The staking is further active and the node is not down. It seems that you can´t connect to the node. Can you share the error log of Firefly? Do you use a VPN or special firewall?
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by mav2022** [comment]
Hey Mike, this is what I´ve found…

Thu, 24 Feb 2022 13:25:37 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 24 Feb 2022 13:25:12 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 24 Feb 2022 12:13:03 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 24 Feb 2022 12:09:53 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 24 Feb 2022 12:06:53 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 24 Feb 2022 12:04:14 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 24 Feb 2022 08:27:44 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 24 Feb 2022 08:26:44 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 21:35:31 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 21:23:08 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 21:19:49 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 21:17:50 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 21:16:08 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 21:15:18 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 21:12:29 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 20:51:48 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 20:50:58 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 20:48:48 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 20:17:53 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 20:17:03 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:51:46 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:50:46 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:43:44 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:42:54 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:41:24 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:40:44 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:39:34 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:38:45 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:38:35 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:38:25 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:37:14 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:36:04 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:32:54 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:05:17 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:04:18 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:04:08 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:02:58 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 19:01:37 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 15:15:34 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 15:15:04 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 15:14:13 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 15:09:19 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 15:07:28 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:07:18 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:07:08 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:06:58 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:06:48 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:06:38 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:06:28 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:06:23 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 15:06:18 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 15:06:18 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:06:17 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:05:51 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:05:51 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:05:41 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:05:41 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:05:36 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:05:32 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:05:31 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:05:21 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:05:21 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:05:11 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 23 Feb 2022 15:05:11 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:05:02 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:04:51 GMT
ClientError: `Response error with status code 526: `

Wed, 23 Feb 2022 15:04:50 GMT
ClientError: `Response error with status code 526: `

Sat, 19 Feb 2022 13:45:19 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 17 Feb 2022 20:25:13 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 17 Feb 2022 17:12:32 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 17 Feb 2022 16:53:00 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 17 Feb 2022 16:52:41 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 17 Feb 2022 14:57:13 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 17 Feb 2022 13:19:07 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Thu, 17 Feb 2022 00:44:14 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 16 Feb 2022 22:49:44 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 16 Feb 2022 22:13:50 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 16 Feb 2022 21:00:14 GMT
ClientError: `error sending request for url (https://mainnet-node.tanglebay.com/api/v1/info): operation timed out`

Wed, 16 Feb 2022 20:58:50 GMT
ClientError: `error sending request for url (https://mainnet-node.tanglebay.com/api/v1/info): operation timed out`

Wed, 16 Feb 2022 20:53:20 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Wed, 16 Feb 2022 20:53:05 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 15 Feb 2022 20:53:12 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 15 Feb 2022 20:53:02 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 15 Feb 2022 17:56:09 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 15 Feb 2022 16:15:11 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 14 Feb 2022 23:38:25 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 14 Feb 2022 21:29:31 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 14 Feb 2022 21:20:24 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 14 Feb 2022 19:19:02 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 14 Feb 2022 18:06:34 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 14 Feb 2022 15:05:25 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 14 Feb 2022 15:01:05 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 14 Feb 2022 14:57:55 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 14 Feb 2022 14:53:45 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 14 Feb 2022 14:51:08 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Mon, 14 Feb 2022 14:50:55 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Fri, 11 Feb 2022 14:10:38 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Fri, 11 Feb 2022 14:04:37 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Fri, 11 Feb 2022 13:14:18 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Fri, 11 Feb 2022 11:11:28 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Tue, 08 Feb 2022 10:56:33 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Sun, 06 Feb 2022 19:25:25 GMT
ClientError: `Response error with status code 520: `

Fri, 04 Feb 2022 21:56:30 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Fri, 04 Feb 2022 20:02:08 GMT
ClientError: `Response error with status code 404: 404 page not found
`

Gesendet von Mail für Windows

Von: MiKeF05
Gesendet: Mittwoch, 23. Februar 2022 19:20
An: iotaledger/firefly
Cc: mav2022; Comment
Betreff: Re: [iotaledger/firefly] FIREFLY: SOMETHING WHEN WRONG! / NETWORKDISCONNECT (Discussion #2417)

The staking is further active and the node is not down. It seems that you can´t connect to the node. Can you share the error log of Firefly? Do you use a VPN or special firewall?
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by MiKeF05** [comment]
Have you already tried to create and restore your profile?

**Comment by mav2022** [comment]
Hey Mike, yes I restored my Profile. Godd New´s-since yesterday without any interupts, the staking is runnig as usual…
So I hope it continues like this.
ThanX 4 your help..
Regards
Maverick

Gesendet von Mail für Windows

Von: MiKeF05
Gesendet: Samstag, 26. Februar 2022 09:16
An: iotaledger/firefly
Cc: mav2022; Comment
Betreff: Re: [iotaledger/firefly] FIREFLY: SOMETHING WHEN WRONG! / NETWORKDISCONNECT (Discussion #2417)

Have you already tried to create and restore your profile?
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
